pkg/security/config/config.go

```diff
@@ -220,9 +223,11 @@ func NewConfig(cfg *config.Config) (*Config, error) {
 		EventStreamBufferSize:              coreconfig.Datadog.GetInt("runtime_security_config.event_stream.buffer_size"),
 
 		// runtime compilation
-		RuntimeCompilationEnabled:       coreconfig.Datadog.GetBool("runtime_security_config.runtime_compilation.enabled"),
-		RuntimeCompiledConstantsEnabled: coreconfig.Datadog.GetBool("runtime_security_config.runtime_compilation.compiled_constants_enabled"),
-		RuntimeCompiledConstantsIsSet:   coreconfig.Datadog.IsSet("runtime_security_config.runtime_compilation.compiled_constants_enabled"),
+		RuntimeCompilationEnabled:            coreconfig.Datadog.GetBool("runtime_security_config.runtime_compilation.enabled"),
+		RuntimeCompiledConstantsEnabled:      coreconfig.Datadog.GetBool("runtime_security_config.runtime_compilation.compiled_constants_enabled"),
+		RuntimeCompiledConstantsIsSet:        coreconfig.Datadog.IsSet("runtime_security_config.runtime_compilation.compiled_constants_enabled"),
+		NetworkProcessEventMonitoringEnabled: coreconfig.Datadog.GetBool("runtime_security_config.event_monitoring.network_process.enabled"),
```

Comment:
just wondering if should move this to a more common key instead of using the `runtime_security_config` key ? like `system_probe_config.event_monitoring` or just `event_monitoring` ?

Reply:
Changed them to be under `event_monitoring_config`, e.g. `event_monitoring_config.network_process.enabled`